lib/gi/configuration.rb

```diff
@@ -8,8 +8,6 @@
 
 module GI
   class Configuration < Common::Client::Configuration::REST
-    self.read_timeout = Settings.gids.read_timeout || 1
```

Comment:
We cannot allow you to increase the timeout back to 15s  for production. If you feel the need to override this for staging you may  do it [here](https://github.com/department-of-veterans-affairs/devops/blob/master/ansible/deployment/config/vets-api/staging-settings.local.yml.j2#L72)

Why:
https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/pull/17/files?short_path=7e8cf21#diff-7e8cf218cb2da0710948af2719b5e320